<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1662</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>40</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <property name="leftMargin">
       <number>30</number>
      </property>
      <property name="topMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>30</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>650</width>
          <height>150</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel
{
    color: rgb(255, 178, 42);
    font: bold 80px;
}</string>
        </property>
        <property name="text">
         <string>Select judge:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="selectJudgeComboBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>90</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>90</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QComboBox
{
border: 1px solid gray;
border-radius: 3px;
padding: 1px 18px 1px 3px;
background-color:rgb(255, 178, 42);
color: white;
	font: 70px &quot;MS Shell Dlg 2&quot;;
}
QComboBox QAbstractItemView{background: rgb(255, 178, 42);}

QComboBox::item:selected
{
    background-color: rgb(255, 190, 50);
    color: rgb(0, 0, 0);
}
QComboBox::drop-down {
    width: 70px;
}

QComboBox::disabled
{
    background-color:  rgb(255, 190, 50);
}</string>
        </property>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Judge 1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Judge 2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Judge 3</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="lockUnlockButton">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>140</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>140</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
color: white;
background-color: rgb(255, 178, 42);
border-radius: 10px;
font: 75 24pt &quot;MS Shell Dlg 2&quot;;
}

QPushButton:hover:!pressed
{
background-color: rgb(255, 190, 50);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="images.qrc">
          <normaloff>:/img/img/lock.png</normaloff>:/img/img/lock.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>200</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label1">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>410</width>
            <height>100</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>100</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel
{
color: rgb(255, 178, 42);
font: 70px &quot;MS Shell Dlg 2&quot;;
}</string>
          </property>
          <property name="text">
           <string>Fight status:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="fightStatusLabel">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>410</width>
            <height>100</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>100</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel
{
color: rgb(255, 178, 42);
font: 70px &quot;MS Shell Dlg 2&quot;;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>30</number>
      </property>
      <property name="leftMargin">
       <number>30</number>
      </property>
      <property name="topMargin">
       <number>30</number>
      </property>
      <property name="rightMargin">
       <number>30</number>
      </property>
      <property name="bottomMargin">
       <number>30</number>
      </property>
      <item>
       <widget class="QPushButton" name="competitor1Button">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>10000000</width>
          <height>10000000</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>50</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
border-image: url(:/img/img/redButton.png);
	color: rgb(235, 113, 113);

}

QPushButton:pressed
{
border-image: url(:/img/img/redButtonPressed.png);
}</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="competitor2Button">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>10000000</width>
          <height>10000000</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>50</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
	border-image: url(:/img/img/blueButton.png);
	color: rgb(139, 178, 239);
 }


QPushButton:pressed
{
border-image: url(:/img/img/blueButtonPressed.png);
}</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
